### PR TITLE
Update to Ember CLI 2.8 and fix failing tests in beta and canary

### PR DESCRIPTION
**Pull request by BrianSipple**
Ever since Ember CLI 2.8 was released, I've noticed that our project (currently on 2.7) has been failing in CI for both "beta" (2.9) _and_ "release" configurations. (Side note: `ember-try` is amazing.)

This can be seen across the builds for the last few PRs (see: the four previous builds from [this point](https://circleci.com/gh/cball/justa-table/112)).

To remedy this, I updated the project to 2.8, ran `ember-init` to resolve any discrepancies, and went about fixing things. Outside of some minor syntax changes, the main changes regarding functionality...
- needing to [use `window.getComputedStyle` for element measurement after render](https://github.com/cball/justa-table/compare/update-version-and-fix-tests?expand=1#diff-45ae4a0e321f20323a82fe5646401cbeR383)
- correcting the way we were [addressing a](https://github.com/cball/justa-table/compare/update-version-and-fix-tests?expand=1#diff-8c21af5cea5e372eb1f173c89fb08c0cR51) [component's `classNames`](https://github.com/cball/justa-table/compare/update-version-and-fix-tests?expand=1#diff-de9a93fc4113a51e044f280e5f066da2R429)

So far, so green. I'll update and rebase #111 and #110 if all goes well here.

### CODE REVIEW

File: tests/integration/components/justa-table-test.js [hunk 1]
```diff
@@ -370,12 +371,16 @@ test('recalculates fixed column width', function(assert) {
 
     {{/justa-table}}
   `);
+
   return wait().then(() => {
+    const columnsElem = getNode(this).querySelector('.fixed-table-columns-wrapper');
+
     // this currently includes 1 px border from the appearance css. not ideal.
-    assert.equal(this.$('.fixed-table-columns-wrapper').width(), 151);
+    assert.equal(parseInt(window.getComputedStyle(columnsElem).width), 151);
```
Comment (cball) [inline tests/integration/components/justa-table-test.js:379]: 👍  on moving to this format. I imagine anywhere we get the width should move to this style, but baby steps. 😄

File: tests/helpers/get-node.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/**
+ * Get the actual DOM node of of the component rendered during an integration
+ * test, which, initially, will have a parent div wrapping it.
+ *
+ * By getting direct access to the aforementioned node, then, we
+ * this utility makes it easy for us to jump right into using the DOM API.
+ */
+export default function getNode(context = this) {
+  return context.$()[0].firstElementChild;
+}
```
Comment (cball) [inline tests/helpers/get-node.js:11]: 👍

File: tests/acceptance/table-rerenders-when-rows-change-test.js [hunk 3]
```diff
@@ -1,24 +1,15 @@
...
+moduleForAcceptance('Acceptance | table rerenders when rows change');
 
 test('basic table properly re-renders when rows are removed', function(assert) {
   visit('/examples/basic-table');
 
   andThen(() => {
-    let rows = $('.table-columns tr').length;
-    let headers = $('.table-columns th').text().trim();
+    let rows = Ember.$('.table-columns tr').length;
+    let headers = Ember.$('.table-columns th').text().trim();
```
Comment (cball) [inline tests/acceptance/table-rerenders-when-rows-change-test.js]: Thoughts on changing `import jQuery from 'jquery'` above vs updating this to `Ember.$`? If you're thinking it's more clear that `Ember.$` does not reference the window scope, I'm good with that.

**Review by cball — APPROVED** [review]
Looks great!

File: tests/acceptance/table-rerenders-when-rows-change-test.js (hunk 3 above)
Comment (BrianSipple) [reply tests/acceptance/table-rerenders-when-rows-change-test.js]: All for it! I'll make that change real quick.

**Comment by BrianSipple** [comment]
@cball just updated and rebase around your `import jQuery` suggestion. Should be good to go!

**Pull request merged**